R/build-home-citation.R

```diff
@@ -24,22 +27,25 @@ data_home_sidebar_citation <- function(pkg = ".") {
   list_with_heading(citation, "Citation")
 }
 
-data_citations <- function(pkg = ".") {
+data_citations <- function(pkg = ".", encoding = "UTF-8") {
   pkg <- as_pkgdown(pkg)
-  cit <- read_citation(pkg$path)
+  cit <- read_citation(pkg$path, encoding)
```

Comment:
Writing R Extensions says:

> The CITATION file is parsed as R code (in the package’s declared encoding, or in ASCII if none is declared)

(see https://cran.r-project.org/doc/manuals/r-release/R-exts.html#CITATION-files)

So shouldn't `data_citations()` just call `read_citation()` with the encoding derived from `pkg$desc`?

Reply:
I don't really want to get into supporting (and hence testing for) any arbitrary encoding. I think UTF-8 is adequate